src/Atmos/Model/turbulence.jl

```diff
@@ -104,11 +104,14 @@ end
 
 function turbulence_tensors(m::ConstantViscosityWithDivergence,
     state::Vars, diffusive::Vars, aux::Vars, t::Real)
-
+  FT = eltype(state)
+  k̂ = aux.orientation.∇Φ / norm(aux.orientation.∇Φ)
```

Comment:
Should this be
```
k̂ = vertical_unit_vector(orientation::Orientation, aux::Vars)
```
from `orientation.jl`? In which case, you'll need to pass in the `AtmosModel`.

If so, please repeat this elsewhere, where needed.

Reply:
Does this work as intended with Spherical orientation?

Reply:
Can we move this whole computation to the CFL computation? The reason I suggest this is that we changing the interface and computation for just the CFL (which isn't called at every time step but only occasionally).

Reply:
The reason this is a concern really is that we might kill our register usage on the GPU doing stuff that isn't required.

Reply:
@jkozdon noted, I will attempt to move this into the CFL computation itself as you suggest.